client/views/editor/editor.js

```diff
@@ -7,15 +7,52 @@ Template.Editor.helpers({
             mode: "text/x-c++src",
             lineWrapping: true,
             cursorHeight: 1.5,
+            theme: "lesser-dark",
         }
     },
 
-    "editorCode": function() {
-        return "Code to show in editor";
-    },
+});
 
-    "getEditorText": function() {
-        return Session.get("varName");
-    }
 
-});
+Template.Editor.events({
+    'click #compile':function(event, template){
+        event.preventDefault();
+        // get property of editor
```

Comment:
Too many inline comments, please drop them and add a general comment for the function following this syntax: http://usejsdoc.org/about-getting-started.html#adding-documentation-comments-to-your-code